subscription-manager.spec

```diff
@@ -1287,12 +1287,10 @@ find %{buildroot} -name \*.py* -exec touch -r %{SOURCE0} '{}' \;
 %dir %{_datadir}/cockpit/subscription-manager
 %{_datadir}/cockpit/subscription-manager/index.html
 %{_datadir}/cockpit/subscription-manager/index.min.js.gz
-%{_datadir}/cockpit/subscription-manager/subscriptions.css
 %{_datadir}/cockpit/subscription-manager/index.css
 %{_datadir}/cockpit/subscription-manager/manifest.json
 %{_datadir}/cockpit/subscription-manager/po.*.js
 %{_datadir}/cockpit/subscription-manager/po.js
-%{_datadir}/cockpit/subscription-manager/node_modules/*
```

Comment:
There is something wrong with this diff. Master branch includes in this section `%if %use_cockpit ... %endif` following line: `%dir %{_datadir}/cockpit/subscription-manager/node_modules`. I added this line 26.6.2020:

https://github.com/candlepin/subscription-manager/pull/2287

You created your feature branch earlier and it is reason, why we see failing builds of tito. Can you please rebase and then remove line with: `%dir %{_datadir}/cockpit/subscription-manager/node_modules`? I'm sure that it will solve the issue with failing tito build.